### ISSUE TITLE
a11y improvements for footer

### ISSUE THREAD

**Issue by maria-jon** [openaq/openaq.org#1127]
Footer accessibility can improve in a few places: 

## Footer

- Form missing labels  
- Social links missing text label & SVG alts/aria-hidden  
- Section tags are missing headings (replace with other elements?)
- Navs missing aria-label for clearer landmarks
